src/module.jl

```diff
@@ -0,0 +1,19 @@
+setpolymodule!(m::JuMP.Model, pm::Module) = setpolymodule!(getpolydata(m), pm)
+setpolymodule!(data::PolyData, pm::Module) = data.polymodule = pm
+
+getpolymodule(m::JuMP.Model) = getpolymodule(getpolydata(m))
+function getpolymodule(data::PolyData)
+    if isnull(data.polymodule)
+        return DefaultModule
+    end
+    get(data.polymodule)
+end
+
+module DefaultModule
+import JuMP, PolyJuMP, MultivariatePolynomials
+function createpoly(m::JuMP.Model, p::Union{PolyJuMP.Poly{false, :Default}, PolyJuMP.Poly{false, :Classic}}, category::Symbol)
```

Comment:
There is a third `category::Bool` argument that needs to be handled, see [here](https://github.com/JuliaOpt/SumOfSquares.jl/blob/c50dd9ce5c3d8c3b6d8a2951953bac4b1adde8c3/src/variable.jl#L27-L32)